plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -226,6 +226,12 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 		return false;
 	}
 
+	// Set texture size limit based on supported feature level.
+	if (level < D3D_FEATURE_LEVEL_11_0)
+		m_d3d_texsize = 8192;
+	else
+		m_d3d_texsize = 16384;
```

Comment:
Would it be better to use `D3D10_REQ_TEXTURE2D_U_OR_V_DIMENSION` and `D3D11_REQ_TEXTURE2D_U_OR_V_DIMENSION`?

Reply:
Cool, much better indeed.